### ISSUE TITLE
Provide GitHub Action which add-on creators can use to opt into Ajour Catalogue

### ISSUE THREAD

**Issue by Feichtmeier** [casperstorm/ajour#237]
So it might look like curse gaming and Overwolf are starting to complain about other WoW add-on managers.

I thought it should be possible to create a github action which add-on creators can use to create pull request in this repository automatically to be included in an Ajour or new repository catalogue.

Most add-ons are developed on GitHub already. Actually there is no need for curse gamings online catalogue.

It would be cool if they would not need to leave GitHub to make their add-ons discoverable for users.

Wdyt?

**Comment by casperstorm** [comment]
I have a statement here where i talk a little bit about the whole situation with Overwolf: https://github.com/casperstorm/ajour/issues/225.

TL;DR is that eventually we (the creators of third party addon managers) will work towards a open source platform were developers can submit their addon to.

**Comment by Feichtmeier** [comment]
Great to hear that @casperstorm 
I think it should be possible to use GitHub as a platform. I mean, if this already exist, and you can even filter github for WoW addons ( https://github.com/topics/wow-addon ) why create yet another platform ? The developers are mostly here anyways :)
